.github/workflows/CI-MATs.yml

```
@@ -0,0 +1,106 @@
+      - name: Post-Clean
+        id: post
+        run: make -j1 -f Makefile clean || true ;
+        if: ${{ always() }}
+
+  MATS_STATUS:
+    needs: MATS
+    runs-on: ubuntu-latest
+    if: ${{ !cancelled() }}
+    outputs:
+      passedMATs: ${{ steps.check_status.outputs.mats_success }}
+      didBUILD: ${{ needs.check_build.outputs.should_run }}
```

actionlint (1.7.4)

property "check_build" is not defined in object type {mats: {outputs: {mats_status: string}; result: string}}

(expression)